### DISCUSSION TITLE
footer text  not customized

### DISCUSSION THREAD

**Discussion by ibrahimalkhalafalnimer** [laravel/nova-issues#3810] [Q&A] [answered]
hello  ... what about footer  custimization in laravel nova i dont have customization until i extract laravel nova outside the vendor file and working with it as package 
i need the footer to be customized like the pervious versions from laravel nova

**Comment by crynobone** [comment, marked as answer]
See <https://nova.laravel.com/docs/4.0/installation.html#customizing-nova-s-footer>
